src/Microsoft.Framework.SecretManager/Program.cs

```diff
@@ -57,9 +58,13 @@ public int Main(string[] args)
             try
             {
                 var app = new CommandLineApplication();
+                app.FullName = "DNX UserSecrets";
                 app.Name = "user-secret";
                 app.Description = "Manages user secrets";
 
+                var assembly = typeof(Program).GetTypeInfo().Assembly;
+                app.ShortVersionGetter = () => assembly.GetName().Version.ToString();
```

Comment:
Okay, great, thanks @Eilon, I had wondered if that was the case with some external input. 

Does that mean that the above implementation (pulling the AssemblyInformationalVersion from the attribute) is okay? 

I have this change complete and I can check that in. I'll do that shortly in the event that this is correct and you folks can merge.